sqldelight-studio-plugin/src/test/kotlin/com/squareup/completion/CompletionTests.kt

```diff
@@ -4,17 +4,20 @@ import com.google.common.truth.Truth.assertThat
 import com.intellij.codeInsight.completion.CompletionType
 import com.intellij.testFramework.fixtures.LightPlatformCodeInsightFixtureTestCase
 import com.squareup.sqldelight.intellij.lang.SqlDelightFileViewProvider
+import com.squareup.sqldelight.types.SymbolTable
 
 class CompletionTests : LightPlatformCodeInsightFixtureTestCase() {
   override fun getTestDataPath() = "testData/completion"
 
   val filesToLoad = arrayOf(
-      "src/main/sqldelight/com/sample/Test1.sq",
-      "src/main/sqldelight/com/sample/Test2.sq"
+      "main/sqldelight/com/sample/Test1.sq",
+      "main/sqldelight/com/sample/Test2.sq",
+      "main/sqldelight/com/sample/Test3.sq"
```

Comment:
the `LightPlatformCodeInsightFixtureTestCase` implementation already has a `src` directory it puts everything in